test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex routing url
+   Given Minishift has state "Running"
+     When status code of HTTP request to "Routing" is equal to "200"
+     Then body of HTTP request to "Routing" contains "ok"
```

Comment:
That does not make sense here, or? Is this a cut&paste error from the health check scenario?